docs/overview/office-add-in-availability.md

```diff
@@ -38,7 +38,9 @@ To work as expected, your Office Add-in might depend on a specific Office host,
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.7</a><br>
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.8</a><br>
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.9</a><br>
-        - <a href="/office/dev/add-ins/reference/requirement-sets/dialog-api-requirement-sets">DialogApi 1.1</a></td>
+        - <a href="/office/dev/add-ins/reference/requirement-sets/dialog-api-requirement-sets">DialogApi 1.1</a><br>
+        - <a href="/office/dev/add-ins/reference/requirement-sets/dialog-api-requirement-sets">ImageCoercion 1.1</a><br>
```

Comment:
Need to update the target URL for every `ImageCoercion 1.#` hyperlink in this article to point to the ImageCoercion requirement sets article (instead of to the Dialog API requirement sets article).